compiler/qsc_frontend/src/compile/preprocess.rs

```diff
@@ -17,7 +17,7 @@ mod tests;
 #[derive(PartialEq, Hash, Clone, Debug)]
 pub struct TrackedName {
     pub name: Rc<str>,
-    pub namespace: Rc<str>,
+    pub namespace: Vec<Rc<str>>,
```

Comment:
I'm a little confused as to when we use the new abstract types to track a namespace path, versus still using a list of strings.